src/index.js

```diff
@@ -12,6 +12,12 @@ import LineStream from './line-stream';
 import ParseStream from './parse-stream';
 import Parser from './parser';
 
+export {
```

Comment:
http://stackoverflow.com/questions/33505992/babel-6-changes-how-it-exports-default#33725725